CRM/Utils/System/DrupalBase.php

```diff
@@ -824,4 +824,18 @@ public function getContactDetailsFromUser($uf_match):array {
     return $contactParameters;
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function modifyStandaloneProfile($profile, $params):string {
+    $config = CRM_Core_Config::singleton();
+    $urlReplaceWith = 'civicrm/profile/create&amp;gid=' . $params['gid'] . '&amp;reset=1';
+    if ($config->cleanURL) {
```

Comment:
In drupal 8 cleanURL always gives 0 even though you can't turn off clean urls in drupal 8 so should always give 1, but that's pre-existing and the link works anyway since drupal figures it out. I'd leave that as a separate issue.

Reply:
That sounds good.